consistency/test.js

```diff
@@ -0,0 +1,21 @@
+const check = require("./check");
+
+const item = (hash) => ({
+    _id: "id",
+    time: "time",
+    query: "query",
+    hash
+});
+
+it("should return a hash if it is not in the item", () => {
+    expect(typeof check(item(undefined))).toBe("string");
+});
+
+it("should return false when the hash does not match", () => {
```

Comment:
is this a valid test for not matching hash? Isnt this more like a null hash? Even empty string generates "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855" on SHA-256

Reply:
the thing being hashed is `id + time + query`

this test is just checking that the hash string (stored in the item) is different than the one computed from the values in the item.

Reply:
ok fair